src/peer-store/index.js

```diff
@@ -83,6 +85,9 @@ class PeerStore extends EventEmitter {
       ...this.metadataBook.data.keys()
     ])
 
+    // Remove self peer if present
```

Comment:
What do you think on this?
For several reasons, we should not return self on peers. We use it widely, for listing peers in IPFS/daemon, for autoDial, ... Users should also be using it, so I think this should not be returned in peers.

Reply:
Yeah we should exclude it when not getting it specifically. Self is not a peer.